content/en/docs/tasks/service-catalog/install-service-catalog-using-helm.md

```diff
@@ -94,8 +94,13 @@ kubectl create clusterrolebinding tiller-cluster-admin \
 Install Service Catalog from the root of the Helm repository using the following command:
 
 ```shell
```

Comment:
I like the idea of this. I recommend using [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) to let the reader switch (with Helm v3 as the default).

Reply:
@sftim that make sense I'll update it accordingly.

Reply:
@sftim @jimangel updated